extensions/aws/s3/S3WrapperBase.cpp

```diff
@@ -68,13 +75,13 @@ void S3WrapperBase::setCannedAcl(Aws::S3::Model::PutObjectRequest& request, cons
   request.SetACL(CANNED_ACL_MAP.at(canned_acl));
 }
 
-std::string S3WrapperBase::getExpiryDate(const std::string& expiration) {
+std::pair<std::string, std::string> S3WrapperBase::getExpirationPair(const std::string& expiration) {
```

Comment:
even though it is only used twice, we could create a `struct` (given how both members of the pair are of the same type)

Reply:
Fixed in [ad6ee95](https://github.com/apache/nifi-minifi-cpp/pull/970/commits/ad6ee955fc776b4816c32b56a802e5a5e30fe41c)